### ISSUE TITLE
Multiselect does not trigger the "change" event on the original <select>

### ISSUE THREAD

**Issue by Unibozu** [yanickrochon/jquery.uix.multiselect#75]
Hi,

Could not find any information about it on the documentation or the wiki. From what I have seen when using your widget, it does not trigger the change event like most multiselect/combo boxes do.

Which is quite convenient : the underlying JS code does not have to rely on the fact that a jQuery plugin is used to function properly.

In my own code, I used the custom event "multiselectChange" but I think this would be more convenient to be able to bind this native event.

Cheers,
Clement

**Comment by yanickrochon** [comment]
Yes, I am aware of this. This was going to be in the next push.... whenever I can manage to free myself of other projects and finally fixed most, if not all, the currently opened issues!

As for now, you can modify the code quite easily by changing the line [35](https://github.com/yanickrochon/jquery.uix.multiselect/blob/master/js/jquery.uix.multiselect.js#L35) of the source code and replacing the value `"multiselectChange"` to `"change"`. This behavior is from a pre-beta release that slipped through the cracks.

See also #58. Also affects #20.
